src/chatlist.rs

```diff
@@ -366,7 +366,9 @@ impl Chatlist {
         let mut lastcontact = None;
 
         let lastmsg = if let Ok(lastmsg) = Message::load_from_db(context, lastmsg_id).await {
-            if lastmsg.from_id != DC_CONTACT_ID_SELF && chat.typ == Chattype::Group {
+            if lastmsg.from_id != DC_CONTACT_ID_SELF
+                && (chat.typ == Chattype::Group || chat.typ == Chattype::Mailinglist)
```

Comment:
maybe sth. as Chattype::is_multiuser() makes sense here, there are only a few more places where that would be needed, so not sure if that makes a lot of sense.